tests/schemas/test_user_schema.py

```diff
@@ -24,6 +25,23 @@ def test_unique_email(self, empty_test_db, create_join_staff):
             f"A user with email '{user.email}' already exists"
         ]
 
+    def test_role_enum_validation(self):
+        user_schema = UserSchema()
+        payload = {"role": 6}
+        validation_error = user_schema.validate(payload)
+        assert "role" in validation_error
+        assert validation_error["role"] == ["Invalid role"]
```

Comment:
Although this works now, a role with the value `6` could conceivably be added in the future, which would break this test.
I'd prefer to see either an extremely unlikely int, a string, or both here. Perhaps something like this:

```suggestion
        payload = {"role": 999}
        validation_error = user_schema.validate(payload)
        assert "role" in validation_error
        assert validation_error["role"] == ["Invalid role"]
        
        payload = {"role": "STAFF"}
        validation_error = user_schema.validate(payload)
        assert "role" in validation_error
        assert validation_error["role"] == ["Invalid role"]
        
```

I'm also not sure we need to be testing for the specific ValidationError message, but we can leave that in for now, since there's another test that also raises a ValidationError for `role`.